core/src/main/kotlin/org/evomaster/core/problem/rest/RestActionBuilderV3.kt

```diff
@@ -538,7 +538,7 @@ object RestActionBuilderV3 {
                         if (it !is String)
                             LoggingUtil.uniqueWarn(log, "an item of enum is not string (ie, ${it::class.java.simpleName}) for a property whose `type` is string and `name` is $name")
                         it.toString()
-                    } as MutableList<String>).apply { add("EVOMASTER") })
+                    } as MutableList<String>).apply {})
```

Comment:
why removing this?

Reply:
Hi @arcuri82, I'm uncertain about adding an extra value to the enum, as it could potentially generate invalid data and lead to casting issues. The same concern applies to handling invalid dates. Is there a purpose that I might be overlooking?

Reply:
ah, I see. sending invalid data in REST makes sense (robustness testing), but not when injecting data into a database. Maybe should add boolean input variable to check if invalid data should be added or not

Reply:
Agreed, I will add the boolean.

Reply:
hi @hghianni  i need to refactor that class for some other reasons... so i ll do this. i ll let you know once branch is then merged into master

Reply:
@hghianni you can pull a fix from master (adding of invalid values is controlled by a parameter now)